config/vo.yml

```diff
@@ -3,17 +3,24 @@
 idspace: VO
 base_url: /obo/vo
 
-base_redirect: http://www.violinet.org/vaccineontology
+base_redirect: https://github.com/vaccineontology/VO
 
 products:
-- vo.owl: https://raw.githubusercontent.com/vaccineontology/VO/master/src/VO_merged.owl
+- vo.owl: https://raw.githubusercontent.com/vaccineontology/VO/master/vo.owl
+- cvo.owl: https://raw.githubusercontent.com/vaccineontology/VO/master/CVO/cvo.owl
```

Comment:
There should not be a "CVO" product here, only "VO" products. Unfortunately the automated tests aren't catching that.

Reply:
@jamesaoverton Thanks for pointing it out. I removed it.